Src/DependencyCollector/NetCore/DependencyCollector.NetCore.csproj

```diff
@@ -1,10 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Copyright>Copyright � Microsoft. All Rights Reserved.</Copyright>
+    <Copyright>Copyright © Microsoft. All Rights Reserved.</Copyright>
     <VersionPrefix>2.4.0-beta1</VersionPrefix>
     <Authors>Microsoft</Authors>
-    <TargetFramework>netstandard1.6</TargetFramework>
+    <TargetFrameworks>netstandard1.3;netstandard1.5</TargetFrameworks>
```

Comment:
the package will have following targets: 
* net40 (from the DependencyCollector.Net40 project)
* net45 (from the DependencyCollector.Net45 project)
* netstandard1.3 (from this project)
* netstandard1.5 (from this project)

Since AspNetCore runs on netstandard1.3, then AI SDK for Core must also allow to be installed on 1.3. And we need a way to collect dependencies for this combination.

The reason for netstandard1.5 target is only to show proper application name in EventSource.